src/openMVG/sfm/sfm_data_io_gt.hpp

```diff
@@ -119,35 +126,47 @@ static bool read_Strecha_Camera(const std::string & camName, Pinhole_Intrinsic &
 
 /**
 @brief Reads a set of Pinhole Cameras and its poses from a ground truth dataset.
-@param[in] functorPointer, to the function which can handle the trajectory format. Example: &read_openMVG_Camera
 @param[in] sGTPath, the directory where the camera files are located.
```

Comment:
Some updates. stl/filesystem doesn't manage recursive search with the wildcard function. Do I work on a recursive digger or we just keep simple with a rollback on solutions 1/2 ?